content/en/BestPractices/SecurityMisconfiguration.md

```diff
@@ -0,0 +1,115 @@
+
+As an example in the apache Tomcat web server, open the conf/web.xml file in the directory where Tomcat is installed. For example in Windows 10, the default installation directory was “C:\Program Files (x86)\Apache Software Foundation\Tomcat 9.0”
+
+In the file, `<param-name>listings</param-name>` the `<param-value>false</param-value>` should be set as false, as shown in the code below
+
+````
+<servlet>
+      <servlet-name>default</servlet-name>
+      <servlet-class>org.apache.catalina.servlets.DefaultServlet</servlet-class>
+      <init-param>
+           <param-name>debug</param-name>
+           <param-value>0</param-value>
+      </init-param>
+      <init-param>
+           <param-name>listings</param-name>
+           <param-value>false</param-value>
+      </init-param>
+      <load-on-startup>1</load-on-startup>
+ </servlet>
+
+````
+
+In the Apache web server, create a .htaccess file in the related application directory and add the following lines to the httpd.conf file or replace the existing lines with the following:
+
+````
+<Directory /{YOUR DIRECTORY}>
+   Options FollowSymLinks
+ </Directory>
+
+````
+
+In general directory listing should be disabled by doing configuration changes in the webserver configuration files.  The configuration will be different for different webservers. 
+
+
+## Default Accounts ##
+
+Default accounts could be application accounts, webserver logins, cloud instances, or databases. At each level, there could be accounts that are in use by developers in the testing or staging environment but were not omitted when the software moves to production. Sometimes these accounts also come with default installations which need to be removed by settings. 
+
+To prevent this, keep a track of the accounts created for the development environment and follow a strict process to remove those accounts later. Change default passwords that come with installations. Restrict the exposure of admin pages or webserver logins while the application is in prod. Follow a strong password policy for the account and prevent the usage of easily guessable credentials.
+
+
+## Files/Directories in URL path ##
+
+Sensitive files such as backup files, log files, or DB files may be present in the URL path, and the Dev team may forget to remove these files or directories from the Webroot. While the application is in the development environment, additional directories can be present in the code, which are not required in the production. If these files or directories get committed to the production, attackers could discover this by brute-forcing the URL paths. Depending on the exposed file/directories, sensitive data can be easily recovered by the attacker. 
+
+To prevent this, ensure a check of the URL paths and remove any sensitive file or directory which is strictly for internal usage. Ensure to have a final check before committing the code to the production environment. 
```

Comment:
In these recommendations, the "ensure to" bit isn't really adding much. We can just say "Check the URL paths and remove any sensitive files" and so on.

On the fence about whether these should be a bulleted list, rather than two sentences.